### PR TITLE
chore: update doc

### PR DESCRIPTION
**Pull request by fospring**
## Pre-flight checklist

- [ ] I have read the [Contributing Guidelines on pull requests](https://github.com/near/near-sdk-js/blob/master/CONTRIBUTING.md).
- [ ] Commit messages follow the [conventional commits](https://www.conventionalcommits.org/) spec
- [ ] **If this is a code change**: I have written unit tests.
- [ ] **If this is a new API or substantial change**: the PR has an accompanying issue (closes #0000) and the maintainers have approved on my working plan.

### CODE REVIEW

File: near-sdk-js@2.0.0-diff-1.0.0.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
+# Breaking features from SDK 2.0.0 diff from 1.0.0
```
Comment (ailisp) [inline near-sdk-js@2.0.0-diff-1.0.0.md]: Let's link this documentation from main readme

File: near-sdk-js@2.0.0-diff-1.0.0.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+            reconstructor: UnorderedMap.reconstruct,  // we need to announce reconstructor explicit
+        });
+        if (innerMap === null) {
+            return null;
+        }
+        return innerMap.get(accountId);
+    }
+}
+```
+In [near-sdk-js 2.0.0](https://www.npmjs.com/package/near-sdk-js/v/2.0.0-0) With schemas, this is no longer needed, as the SDK can correctly infer how to decode the Nested Collections:
```
Comment (ailisp) [inline near-sdk-js@2.0.0-diff-1.0.0.md]: also update this to 2.0.0

File: near-sdk-js@2.0.0-diff-1.0.0.md (hunk 1 above)
Comment (fospring) [reply near-sdk-js@2.0.0-diff-1.0.0.md]: ok, added link from README.md

File: near-sdk-js@2.0.0-diff-1.0.0.md (hunk 2 above)
Comment (fospring) [reply near-sdk-js@2.0.0-diff-1.0.0.md]: ok,updated

**Review by ailisp — APPROVED** [review]

**Pull request merged**